test/performance/perf-upsert/src/config.rs

```diff
@@ -40,19 +40,22 @@ pub struct Args {
     pub kafka_topic: String,
 
     /// Whether or not to delete the sources and views before starting
-    #[structopt(long)]
+    #[structopt(long, requires("create-topic"))]
```

Comment:
I don't think preserve-source should be required with the other ones

Reply:
I wasn't sure -- I just wanted to demonstrate the code -- feel free pull my commit and then trash all of it. Or just force-push over it.

Reply:
but thank you for figuring out the requires bit! very very helpful